evaluation/user-feedback.mdx

```diff
@@ -1,3 +1,4 @@
+{/* TODO: this page is basically the same thing as monitoring/collect-user-feedback. not sure what to do with it */}
```

Comment:
Good point, let's remove this page entirely and keep the `monitoring/collect-user-feedback` but only after we merge these two sections into one (rename `Instructions` to `Capturing Annotations` and use the example code from the evaluation page because it shows how to use other methods like `log`.

Note: Make sure to update all links to this page in the docs to reflect this change, and create a task for Melville to update links (if there are any) in the app too.

<img width="1496" alt="image" src="https://github.com/baserun-ai/docs/assets/1614514/a47d0649-6795-4d9c-879c-5ebd07ded054">